contrib/gorm.io/gorm/example_test.go

```diff
@@ -0,0 +1,41 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package gorm_test
+
+import (
+	"database/sql"
+	"github.com/jackc/pgx/v4/stdlib"
+	gorm2 "gopkg.in/DataDog/dd-trace-go.v1/contrib/gorm.io/gorm"
```

Comment:
This needs to be aliased as `gormtrace` and the imports need to ordered into 3 groups:
1. Standard library
2. Local (`dd-trace-go`) packages
3. External packages

You will notice this pattern in the other files.